classes/character.js

```diff
@@ -58,22 +63,55 @@ ClassManager.create('Character', function(game) {
        * Returns true if the player's screen position matches that of the game position (scaled)
        */
       isMoving: function() {
-         return !(this.x === this.position.x * C.TILE_SIZE && this.y === this.position.y * C.TILE_SIZE);
+         return !this.isAttacking() && !(this.x === this.position.x * C.TILE_SIZE && this.y === this.position.y * C.TILE_SIZE);
```

Comment:
With how much we're gonna be using X * c.tile_size, maybe turn it into a conversion helper method?